### PR TITLE
Task/8 create side menu

### PR DESCRIPTION
**Pull request by ranjini-mobile-dev**
#8 closes

### CODE REVIEW

**Comment by santhosh-mobile-dev** [comment]
Rename it as side_menu.dart and class as SideMenu

File: lib/ui/screen/home_drawer.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            ),
+            Text("Name", style: TextStyle(fontSize: 22.0, color: Colors.white),)
+          ],
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
```
Comment (santhosh-mobile-dev) [inline lib/ui/screen/home_drawer.dart]: Why build is returning widget .. according to our coding standards build should not return widget ..

File: lib/ui/widgets/custom_list_tile.dart [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class CustomListTile extends StatelessWidget {
+  final IconData iconData;
+  final String text;
+
+  CustomListTile(this.iconData,this.text);
```
Comment (santhosh-mobile-dev) [inline lib/ui/widgets/custom_list_tile.dart]: Why these properties are not optional .. it’s safe to make them optional .. give attention in call ... when I tell people to make constructor using {}

**Review by santhosh-mobile-dev — CHANGES_REQUESTED** [review]
See inline comment and make changes accordingly

**Comment by santhosh-mobile-dev** [comment]
@darshanaAeione dont approve PR if I asked for changes .. approve only when I approve and you are not having anything to comment ..

**Review by santhosh-mobile-dev — APPROVED** [review]

**Review by darshanaAeione — APPROVED** [review]

**Pull request merged**